.github/workflows/github-cxx-qt-tests.yml

```diff
@@ -88,6 +88,55 @@ jobs:
           token: ${{ secrets.CODECOV_TOKEN }}
           verbose: true
 
+  build-wasm:
```

Comment:
can you add a `name: Ubuntu 22.04 (wasm_32) Qt 6` as a key, then this will appear nicely in the CI workflow